### ISSUE TITLE
Draw an horizontal line on a multibarchart for reference

### ISSUE THREAD

**Issue by Bakachimay** [krispo/angular-nvd3#363]
Hi folks,

I want to draw a line over my multi-bar chart for reference. Does anyone know how to do this ? 

A line like the dashed one here : http://www.mrc-productivity.com/techblog/images/target.png

I am still looking after a solution for this issue too: https://github.com/krispo/angular-nvd3/issues/351

Thanks,

**Comment by csomers** [comment]
I adapted this  stackoverflow answer to get something similar:
http://stackoverflow.com/a/28568763

**Comment by Bakachimay** [comment]
I have a little problem with this, the scale() function doesn't seem to work in angular-nvd3 is it possible ? I had already found solutions in d3.js with scale() but without it never...

**Comment by corina10** [comment]
Has anyone found a solution for this issue yet?

**Comment by iamdevlinph** [comment]
This might be late but hope it helps future viewers:
http://plnkr.co/edit/dc4dHHZUZYQL2moyBEcy?p=preview

The line is manually drawn using d3

**Comment by ErvinLlojku** [comment]
@iamdevlinph Can you help us with the code to scale the horizontal line based on chart width?

**Comment by iamdevlinph** [comment]
Get the width of the chart. Strip off the margins if necessary. Then draw the horizontal line.

If you want the line to be "responsive". You can create a directive that listens to the window resize event. When fired, the directive will redraw the lines based on the new width.

**Comment by iamdevlinph** [comment]
This horizontal line is just a horizontal line. It's not a line chart that uses the same x-axis as the rest of the chart. This line is just a single data. Think of it as an average line.

**Comment by ErvinLlojku** [comment]
@iamdevlinph Thanks for the response. I have done the line width resize based on chart width when window resize event is fired. I was thinking if there was a better way. Thanks 👍

**Comment by iamdevlinph** [comment]
Yeah. That's the only way that I know. Since the line is a fixed element on the page with fixed dimensions.
